{{cookiecutter.app_name}}/{{cookiecutter.app_name}}/database.py

```diff
@@ -33,7 +35,9 @@ def save(self, commit=True):
     def delete(self, commit=True):
         """Remove the record from the database."""
         db.session.delete(self)
-        return commit and db.session.commit()
+        if commit:
+            return db.session.commit()
+        return
```

Comment:
This changes the return type of the function: previously, `delete` would have returned `False` if `commit` was also `False`: now it will return `None`. 

I think the change is better than what was here before, but perhaps you could add a mypy annotation to the function signature to clarify expectations (`def delete(self, commit: bool=True) -> None:`)? 

This reminds me that I should start trying to add in more annotations to this project 😄